exercises/knapsack/.meta/src/reference/java/Knapsack.java

```diff
@@ -0,0 +1,201 @@
+import java.util.ArrayList;
+
+/**
+ * Knapsack item with a weight and value
+ */
+class Item {
+
+    // Weight of the item
+    int weight;
+    // Value of the item
+    int value;
+
+    Item(int itemWeight, int itemValue) {
+        this.weight = itemWeight;
+        this.value = itemValue;
+    }
+    
+}
```

Comment:
Please extract this class

Reply:
in a separate file?

Reply:
Yes, that would be great. We have many exercises and hence reference implementations that use more than one class.

Reply:
Extracted Item into Item.java